docs/go/quickstart/gin-mongo-quickstart.md

```diff
@@ -13,17 +13,15 @@ tags:
 
 A sample url shortener app to test Keploy integration capabilities in Go.
 
-In this sample application we're using 2 dependencies - [Gin](https://gin-gonic.com), mongoDB.
+In this sample application we're using 2 dependencies - [Gin](https://gin-gonic.com), [mongoDB](https://www.mongodb.com/).
 
 ### Pre-requisites
 
 - [Go](https://golang.org/doc/install)
 
-import ServerInstallation from '../../server/server-installation.md'
+## Installation Methods
 
-<ServerInstallation/>
-
-Once you have the Keploy Server running, open the Keploy Console at [http://localhost:8081](http://localhost:8081)
```

Comment:
I dont think deleting these is a good call

Reply:
it's already mentioned in server installation 
![Screenshot 2022-10-09 at 4 11 52 PM](https://user-images.githubusercontent.com/85970385/194752249-152fe84d-b72d-4acc-b891-a18c0e5ecb1f.png)
and @SanskritiHarmukh mam mentioned
We can remove one of the line from documentation in [Gin-Mongo Sample](https://docs.keploy.io/docs/go/quickstart/gin-mongo-quickstart)
![screen shot](https://user-images.githubusercontent.com/74777863/193319951-3185498e-baa2-4bdf-a817-533fe5bb8a8e.png)

Reply:
oh sure sure ! We can lead with this

Reply:
Please review it